bootstrap.sh

```diff
@@ -1 +1,5 @@
 #!/bin/bash
+kubectl apply -f confgiMap.yml
```

Comment:
There is a typo in the filename 'confgiMap.yml'. It should be 'configMap.yml' to match the standard naming convention and ensure the file is correctly applied.